configure.ac

```diff
@@ -16,9 +16,11 @@ AM_PROG_CC_C_O
 AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 
-dnl i18n
-IT_PROG_INTLTOOL([0.35.0])
-GP_I18N
+dnl i18n, inline so autoreconf can detect gettext usage
+AM_GNU_GETTEXT_VERSION([0.23.1])
```

Comment:
I hope this not too new and only relevant when running "autogen.sh".

With the old version, I got errors when running "make" like:
```
*** error: gettext infrastructure mismatch: using a Makefile.in.in from gettext version 0.19 but the autoconf macros are from gettext version 0.22
```

I hope this is OK.

Reply:
I reverted this to the version 0.19 as in Geany, this makes the CI happy.

Reply:
Maybe that's the kind of problem newer versions tried to fix, but if we don't have the issue in Geany we should be able to figure something out.

Reply:
It seems we need to define `AM_GNU_GETTEXT_REQUIRE_VERSION`.

From my tests, it doesn't make a difference if we define it instead of or additionally to `AM_GNU_GETTEXT_VERSION`, Windows build succeeds once it is set.

Yet, I still have no clue why, what's the difference to Geany and what exactly these two options really do :(.

@b4n @techee do you have any clue or do we want to use it this way? I would cleanup the commits then.

Reply:
Oficially, I have no clue about anything related to autotools. And whatever anyone does to make it work is more than enough for getting it merged. :-)